dotcom-rendering/src/components/ManyNewslettersFormFields.tsx

```diff
@@ -33,6 +33,23 @@ const optInCheckboxTextSmall = css`
 	}
 `;
 
+const recaptchaContainerStyle = (
+	visibleRecaptcha: boolean,
+	firstInteractionOccurred: boolean,
+) => css`
+	margin-bottom: ${visibleRecaptcha && firstInteractionOccurred
+		? space[3]
+		: 0}px;
+	padding: ${visibleRecaptcha && firstInteractionOccurred ? space[2] : 0}px;
+	background-color: ${visibleRecaptcha && firstInteractionOccurred
+		? palette.neutral[93]
+		: 'transparent'};
+
+	.grecaptcha-badge {
+		visibility: hidden;
+	}
+`;
```

Comment:
Since both `visibleRecaptcha` & `firstInteractionOccurred` are always used together, could it be cleaner if we just have 1 parameter here which gets the value of `visibleRecaptcha && firstInteractionOccurred`